esp/xslt/esdl2monitor_cassandra.xslt

```diff
@@ -0,0 +1,1122 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Note that this is just a copy of the old esdl2monitor.xslt.  Nothing has changed except the copyright date.